### ISSUE TITLE
Rekognition:DetectText API is failing with a Missing Authentication Token error

### ISSUE THREAD

**Issue by Amplifiyer** [aws/aws-sdk-js-v3#793]
**Describe the bug**
Calling DetectText API of Rekognition fails at the OPTIONS request. Following errors are observed

// console
```
OPTIONS https://rekognition.us-east-1.amazonaws.com/DetectText 403 (Forbidden)
Access to fetch at 'https://rekognition.us-east-1.amazonaws.com/DetectText' from origin 'http://localhost:3000' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

// API response
```xml
<MissingAuthenticationTokenException>
  <Message>Missing Authentication Token</Message>
</MissingAuthenticationTokenException>
```

**SDK version number**
@aws-sdk/client-rekognition@1.0.0-alpha.11

**Is the issue in the browser/Node.js?**
Browser

**Details of the browser/Node.js version**
  Browsers:
    Chrome: 79.0.3945.117
    Safari: 13.0.4

**To Reproduce (observed behavior)**
Make a service call to Rekognition:DetectText

**Expected behavior**
There should be no error.

**Screenshots**

**Additional context**

**Comment by Amplifiyer** [comment]
Exact same error with `Textract:AnalyzeDocument` API call.

**Comment by srchase** [comment]
Making the same request using V2 of the SDK succeeds.  A POST is sent directly, whereas for V3 receives a 403 on the preflight OPTIONS request.

**Comment by srchase** [comment]
Fixed via https://github.com/aws/aws-sdk-js-v3/pull/837

**Comment by trivikr** [comment]
@Amplifiyer verified via IM that the issue is fixed in `@aws-sdk/client-rekognition@1.0.0-alpha.14`
